lms/static/sass/base/_grid-settings.scss

```diff
@@ -7,7 +7,7 @@ $border-box-sizing: false;
 
 
 /* Breakpoints */
-$mobile: new-breakpoint(max-width 320px 4);
-$tablet: new-breakpoint(min-width 321px max-width 768px, 8);
+$mobile: new-breakpoint(max-width 540px 4);
```

Comment:
I agree with @martynjames on this one - @marjev, can you add an additional breakpoint to this list for your case of 321px - 540px and re-instate the original mobile one?

Also, extra bonus points for renaming them to the sustainable system I mentioned above.